docs/how_to/use_toggles.rst

```diff
@@ -0,0 +1,65 @@
+***************************************
+How to: Use toggles
+***************************************
+
+Calling toggles
+==================
+
+Unlike the underlying waffle library which looks up waffles by name,
+toggles should be imported and used directly:
+
+.. code:: python
+
+    from whereever import FLAG_FOR_STUFF, COURSE_FLAG_FOR_STUFF
+
+    FLAG_FOR_STUFF.is_enabled()
+    COURSE_FLAG_FOR_STUFF.is_enabled('course-key-123')
+
+You can get at the state of these toggles by name via the waffle library
+and it will appear to work fine in simple testing. Don't be fooled.
+
+How toggles get their results
+====================================
+
+Obviously since some toggles take into account the user they must have more data
+than the zero arguments of ``is_enabled()``.
+Under the hood toggles use:
+
+* A per-request cache, so a toggle value is stable during the request
+* If needed by this toggle type, the request via ``crum.get_request`` to pass to the underlying waffle code
+
+
+Using toggles you don't want to import
```

Comment:
I like that. I also think it's worth cautioning the possible caller to avoid using toggles in an end-run around the public API since python being python it's easy to do so.

Reply:
Sure. I just recommend keeping things short to not lose focus, for those like me that often just read enough to feel like my question is answered. :)